backend/app/tests/conftest.py

```diff
@@ -1,5 +1,8 @@
 import os
 
+# Import fixtures
+pytest_plugins = ("fixtures.corgi",)
```

Comment:
Also, the location of the fixtures file should be inside the tests folder.

Reply:
I did a quick change on my local. This is what I did:

<img width="1185" alt="2022-03-22_10-16-42" src="https://user-images.githubusercontent.com/8730430/159515909-972b39d6-3dac-4dd7-a5f4-83921d788b0a.png">

Reply:
This was again, copying from cnx-automation... :)